test/cpp/jit/test_gpu.cpp

```diff
@@ -3899,6 +4007,63 @@ void testGPU_FusionZeroDimReduction() {
       aten_output.sub(output).abs().max());
 }
 
+void testGPU_FusionBCastAfterReduce() {
+  torch::jit::fuser::cuda::CudaKernel prog;
+  Fusion& fusion = *prog.fusion_;
+  FusionGuard fg(&fusion);
+
+  const int tidx = 128;
+
+  // Set up your input tensor views
+  TensorView* tv0 = makeDummyTensor(2);
+  fusion.addInput(tv0);
+
+  auto tv1 = sum(tv0, {1});
+  tv1->split(1, tidx);
+  auto tv2 = tv1->rFactor({-2});
+
+  auto tv3 = broadcast(tv1, {false, true});
```

Comment:
same question as earlier, can we use just {true} for the broadcast dimensions?

Reply:
I don't think so.